schema.ts

```diff
@@ -1374,6 +1375,16 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    */
   index?: IndexDefinition;
 
+  /**
+   * The name of the property within {@link ObjectSchemaDefinition.properties} that can be be interpreted as
+   * text representing the state of an item.
+   *
+   * Must be a {@link ValueType.String} property.
+   * TODO(ebo): Unhide this
+   * @hidden
+   */
+  itemStateProperty? : PropertyIdentifier<K>;
```

Comment:
I do not like this name. What is the "state of an item"?  That could mean anything

Reply:
changed to version